Increase client body size for all review apps.

lmontopo opened the pull request:
Review apps consistently run into issues because the default
limit is quite small.

Commit:
Increate client body size for all review apps.

Review apps consistently run into issues because the default
limit is quite small.

hrpatel commented:
Have you tried running this locally to see if the configuration loads?

antevens commented:
+1 assuming it's tested and works

lmontopo commented:
@hrpatel you know i ran it locally just to make sure i didn't get an error. but your comment made me realize i should actually look at the resulting nginx container and make sure that it has the configuraitons i expect... and it doesn't. Which i'm quite confused about. It just doesn't have that extra custom-nginx.conf file i copied over to it... any ideas?

lmontopo commented:
I also get this warning but i disregarded it previously because i wasn't doing anything to `/etc/nginx/conf.d/default.conf`

```
2021/02/16 21:04:15 Contents of /etc/nginx/conf.d/default.conf did not change. Skipping notification 'nginx -s reload'
```

antevens commented:
Nginx does not include all files in the directory unless instructed to do so. you'll have to add an include to the default file.

lmontopo commented:
oh wait i think i know why it wasn't working for me. The docker-compose file doesn't even do anything with the docker file, it seems? Hm. This repo is confusing. I shall read the docs again. 

And @antevens i thought that the default `nginx.conf` container usually had a line like `include /etc/nginx/conf.d/*.conf;` in it?

lmontopo commented:
once the containers were tagged jwilder/nginx-proxy:latest, and once we removed some of our own cert code from the template file, the resulting nginx container did have the config i added, and the main config did include it :) going to merge!
